tests/integration/test_github_workflows.py

```
@@ -6,11 +6,10 @@
 duplicate keys, invalid syntax, and missing required fields.
 """
 
-import os
 import pytest
 import yaml
 from pathlib import Path
-from typing import Dict, Any, List
+from typing import Any, Dict, List
```

Ruff (0.14.6)

`typing.Dict` is deprecated, use `dict` instead

(UP035)